site/src/api/queries/users.ts

```diff
@@ -89,21 +89,12 @@ export const authMethods = () => {
   };
 };
 
-const initialMeData = getMetadataAsJSON<User>("user");
-const meKey = ["me"] as const;
-
-export const me = (queryClient: QueryClient) => {
+export const me = () => {
   return {
-    queryKey: meKey,
-    queryFn: async () => {
-      const cachedData = queryClient.getQueryData(meKey);
-      if (cachedData === undefined && initialMeData !== undefined) {
-        return initialMeData;
-      }
-
-      return API.getAuthenticatedUser();
-    },
-  } satisfies UseQueryOptions<User>;
+    queryKey: ["me"],
+    initialData: getMetadataAsJSON<User>("user"),
```

Comment:
My concern with this approach is that `getMetadataAsJSON` will get recalculated every single render, even though the value is basically a constant embedded in the HTML

This should be safe to extract outside the function